MapboxAndroidDemo/src/global/java/com/mapbox/mapboxandroiddemo/MapboxApplication.java

```diff
@@ -18,10 +18,13 @@
   public void onCreate() {
     super.onCreate();
     setUpPicasso();
+    FirebaseApp.initializeApp(this);
```

Comment:
This can also be if-check for debug builds, otherwise, it prints a lot of failure logs in the logcat.

Reply:
Ok, fixed to 

```java
if (!BuildConfig.DEBUG) {
      FirebaseApp.initializeApp(this);
      setUpCrashlytics();
    }
```